src/test/java/UnitTests.java

```diff
@@ -0,0 +1,99 @@
+import com.google.gson.Gson;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+
+import java.io.File;
+
+public class UnitTests {
```

Comment:
Do not use plurals in class names. 1) it is incorrect according to java coding standard; 2) you can try to run tests using maven: mvn clean test. Nothing will be executed until you change configuration for surefire plugin (maven searches for Test*, *Test by default)